app/src/main/java/com/duycuong/weather/utils/TimeUtils.java

```diff
@@ -11,6 +12,22 @@
     public static String covertTime(String format, long time) {
         SimpleDateFormat simpleDateFormat =
                 new SimpleDateFormat(format, Locale.getDefault());
-        return simpleDateFormat.format(time * 1000);
+        return simpleDateFormat.format(time*1000);
+    }
+
+    public static long ConvertDateToTimeStamp(String ngay){

```

Comment:
ERROR: [MethodNamingConventions] Method names should not start with capital letters
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#MethodNamingConventions